components/Starknet/modules/tools/pages/compatibility.adoc

```diff
@@ -0,0 +1,72 @@
+== Starknet Compatibility Matrix
+
+The Starknet development ecosystem consists of tools, built to work and support the latest version of Starknet. As Starknet evolves, APIs and tools are updated to support the latest features.
+
+Each version of Starknet is usually coupled with an update of all or some of the following:
+
+* Starknet API
+* Cairo
+* Sierra
+
+To support these changes, the Starknet full nodes (Pathfinder and Juno) implement the latest API version and Starknet protocol.
+
+Supporting the version of Starknet are tools for contract development, CLIs and SDKs. The following compatibility matrix provides an overview of the compatibility between Starknet versions and the tools in the ecosystem.
+
+
+[NOTE]
+====
+Currently this page is being manually updated. Newer versions of tools may exist and work with current or older Starknet versions. 
+You are welcome to create a PR and update the version to the newest one you have tested.
+====
+
+== Compatibility Matrix
+
+
+|===
+| | Starknet 0.13.2 | Starknet 0.13.1 | Starknet 0.13.0 
+| RPC Version | 0.7 | 0.7 | 0.6
+| Cairo Version | 2.8.0 | 2.6.0 | 2.4.0
+| Sierra Version | 1.6.0 | 1.5.0 | 1.4.0
+| Pathfinder | 0.14.1 | 0.11.0 | 0.10.0
+| Juno | 0.12.0 | 0.11.0 | 0.8
+|===
+
+== Starknet API
+
+|===
+| | RPC 0.7 | RPC 0.6
+| starknet.js | ≥ 6.11.0 | 5.14.0
+| starknet.py | ≥ 0.21.0 | 0.20.0
+| starknet-rs | ≥ 0.11.0 | 0.10.0
+| starknet.go | ≥ 0.7.0 | 0.6.1
+| starkli | ≥ 0.3.0 | 0.2.0
+| Pathfinder | ≥ 0.11.0 | 0.10.2
+| Juno | ≥ 0.11.0 | 0.8.0
+| Starknet Foundry | ≥ 0.28.0 | 0.19.0
+| starknet-devnet-rs | ≥ 0.1.4 | 0.1.0
```

Comment:
The last Devnet with RPC 0.6 support is [v0.0.2](https://github.com/0xSpaceShard/starknet-devnet-rs/releases/tag/v0.0.2). The first with RPC 0.7 support is [v0.0.4](https://github.com/0xSpaceShard/starknet-devnet-rs/releases/tag/v0.0.4). Between them, release candidates for 0.7 were supported.